### ISSUE TITLE
Incorrect Android Release Build Path 

### ISSUE THREAD

**Issue by btate** [bamlab/fastlane-plugin-cordova#30]
The plugin builds the android apk at 

```
./platforms/android/app/build/outputs/apk/release/app-release.apk
```

The value of `ENV['CORDOVA_ANDROID_RELEASE_BUILD_PATH']` is

```
./platforms/android/build/outputs/apk/android-release.apk
```

The `release` folder in that path is missing.

**Comment by lampi87** [comment]
Same for me.
It works with cordova-android 6.3.0 but not with 6.4.0

**Comment by Almouro** [comment]
Hello guys, a new version has been published as `2.0.0` using the new path! 🎉
